Source/ImPlotWidgets/Private/LineFilledGraphWidget.cpp

```diff
@@ -43,18 +61,27 @@ void SFilledGraphWidget::OnImGuiDraw() const
 		| ImGuiWindowFlags_NoBackground
 		| ImGuiWindowFlags_NoSavedSettings;
 
-	constexpr int32 GraphFlags = ImPlotFlags_NoLegend | ImPlotFlags_AntiAliased;
+	constexpr int32 GraphFlags = ImPlotFlags_AntiAliased;
 
 	constexpr int32 AxisFlags = ImPlotAxisFlags_NoGridLines;
 
 	const FGeometry& AllottedGeometry = ImGuiCustomWidget.Get()->GetTickSpaceGeometry();
 
 	constexpr bool* CloseButtonVisible = nullptr;
 
+	static TArray<bool> ShowLine;
+
 	ImGui::SetNextWindowPos(ImVec2(0, 0), ImGuiCond_Always);
 	ImGui::SetNextWindowSize(ImVec2(AllottedGeometry.GetLocalSize().X, AllottedGeometry.GetLocalSize().Y), ImGuiCond_Always);
 	ImGui::Begin("ImPlot Demo", CloseButtonVisible, WindowFlags);
 
+	for (int32 CurrentArray = 0; CurrentArray < XValueArrays.Num(); ++CurrentArray)
+	{
+		ImGui::SameLine();
+		ShowLine.Add(true);
```

Comment:
Does the OnImGuiDraw function get called every frame? If so I think this will keep adding entries to this array.